LayoutTests/fast/dom/Window/resources/window-properties.js

```diff
@@ -123,7 +119,6 @@ var propertyInfo = [
     ["ProcessingInstruction", "object"],
     ["ProgressEvent", "object"],
     ["Range", "object"],
-    ["Rect", "object"],
```

Comment:
I would also restore all of these, until the functionality actually gets removed.